international/international_script.sh

```diff
@@ -73,4 +73,17 @@ sleep 10
 echo "Running containers:"
 docker ps
 
+# check if the DHIS2 core server is running before proceeding
+echo "Checking if the DHIS2 core server is running..."
+
+while [[ $(curl -v http://localhost:8080 2>&1) != *"Connected to localhost"* ]]; do
```

Comment:
Could we please make this dynamic? Use the `DHIS2_CORE_PORT` env variable and default to 8080 if not set.